plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -241,7 +241,7 @@ public Set<String> getAllTables(String schema)
                 .filter(name -> !name.equals(schemaCollection))
                 .filter(name -> !SYSTEM_TABLES.contains(name))
                 .collect(toSet()));
-        builder.addAll(getTableMetadataNames(schema));
+        builder.addAll(getTableMetadataNames(schemaName));
```

Comment:
Can we rename `schemaName` to `remoteSchemaName`? It is kind of a pity that we don't have `RemoteSchemaName` class (or is it an overkill?).

Reply:
We can, I want to do this, but I prefer not to. 
Mongo history of remoteSchemaName implementation is long and tedious. 
There are many naming inconsistencies in the code.
I think this can be postponed at list till `IdentifierMappingModule` is used for Mongo.

Reply:
> till IdentifierMappingModule is used for Mongo.

Great idea!